security-oauth2/src/main/java/io/micronaut/security/oauth2/endpoint/authorization/response/AuthorizationErrorResponseExceptionHandler.java

```diff
@@ -29,6 +42,26 @@
  */
 @Singleton
 public class AuthorizationErrorResponseExceptionHandler implements ExceptionHandler<AuthorizationErrorResponseException, MutableHttpResponse<?>> {
+    private static final Logger LOG = LoggerFactory.getLogger(AuthorizationErrorResponseExceptionHandler.class);
+    private final RedirectConfiguration redirectConfiguration;
+    private final RedirectService redirectService;
+
+    private final ErrorResponseProcessor<?> errorResponseProcessor;
+
+    /**
+     *
+     * @param redirectConfiguration Redirect Configuration
+     * @param redirectService Redirect Service
+     * @param errorResponseProcessor Error Response Processor
+     */
+    @Inject
+    public AuthorizationErrorResponseExceptionHandler(RedirectConfiguration redirectConfiguration,
```

Comment:
Needs nullability annotations, and the fields too

Reply:
I added `@Nullability` annotations and test via https://github.com/micronaut-projects/micronaut-security/pull/1232/commits/40d1e364693dfaa78db7774201eb806c7df1e9d7